src/python/pants/core/util_rules/environments.py

```diff
@@ -212,17 +213,17 @@ def debug_hint(self) -> str:
 
 
 @dataclass(frozen=True)
-class ResolvedEnvironmentTarget:
+class EnvironmentTarget:
     val: Target | None
 
 
 @dataclass(frozen=True)
-class ResolvedEnvironmentRequest(EngineAwareParameter):
-    f"""Normalize the value into an alias from `[environments-preview].aliases`, such as by
+class EnvironmentRequest(EngineAwareParameter):
```

Comment:
NB: this name conflicts with `engine.environment.EnvironmentRequest`, which is used to request the env vars

Reply:
I don't expect that `EnvironmentRequest` needs to be a part of the "public" API of environments, so I don't think that that will be an issue.